pkg/scheduler/objects/nodesorting.go

```diff
@@ -62,6 +63,16 @@ func (fairnessNodeSortingPolicy) SortNodes(nodes []*Node) {
 	})
 }
 
+func (binPackingNodeSortingPolicy) ScoreNode(node *Node) float64 {
+	// choose most loaded node first
+	return resources.LargestUsageShare(node.GetAvailableResource())
+}
+
+func (fairnessNodeSortingPolicy) ScoreNode(node *Node) float64 {
+	// choose least loaded node first
+	return -resources.LargestUsageShare(node.GetAvailableResource())
+}
+
```

Comment:
our current node sorting algorithm is problematic... it is not caused by this PR, but still want to raise it up for attention. Simply using largest resource share for calculating the score is wrong, we need to consider all the resources. when there is only cpu and memory, memory will be always leading the calculation; but in many cases, cpu is a more expensive resource, our result is suboptimal. 
As a reference, we can look at how this is done in: https://github.com/kubernetes/kubernetes/tree/master/pkg/scheduler/framework/plugins/noderesources. we need something similar for our FAIR & Bin-packing policies. We can have a follow up JIRA for this.

Reply:
Agree the current algorithm needs work, and that we should address in a followup JIRA.

Reply:
+1